src/aleph/web/controllers/main.py

```diff
@@ -82,8 +82,8 @@ async def metrics_json(request: web.Request) -> web.Response:
 
 
 class Metrics(BaseModel):
-    start_date: Optional[float] = None
-    end_date: Optional[float] = None
+    start_timestamp: Optional[float] = None
+    end_timestamp: Optional[float] = None
```

Comment:
Will this change break the frontend account and console pages?

Reply:
This API is not used anywhere that I am aware of. We plan to upgrade the metrics graph on CRN homepages to use it in the future.